packages/webapp/src/components/Map/SelectionHandler/index.jsx

```diff
@@ -48,89 +126,117 @@ export default function PureSelectionHandler({ locations, history, sensorReading
-    } else if (location.type === SENSOR) {
-      history.push(`/${location.type}/${location.id}/readings`);
     } else {
       history.push(`/${location.type}/${location.id}/details`);
     }
   };
 
-  const removeSelect = isTouchDevice()
-    ? {
-        userSelect: 'none',
-        WebkitUserSelect: 'none',
-        msUserSelect: 'none',
-        MozUserSelect: 'none',
-      }
-    : {};
-
-  return locations.map((location, idx) => {
-    const { type, asset, name } = { ...location };
-    let icon = imgMapping(asset, type);
+  return (
+    <div className={classes.container}>
+      <div className={classes.tooltip}>
+        <div className={classes.arrow} />
+        {locations.map((location, idx) => {
+          const { type, asset, name } = { ...location };
+          let icon = imgMapping(asset, type);
 
-    return (
-      <div
-        key={idx}
-        {...(isTouchDevice()
-          ? {
-              onTouchStart: (e) => {
-                e.stopPropagation();
-                handleMouseDown(location, idx);
-              },
-              onTouchEnd: () => handleMouseUp(location),
-            }
-          : {
-              onMouseDown: (e) => {
-                e.stopPropagation();
-                handleMouseDown(location, idx);
-              },
-              onMouseUp: () => handleMouseUp(location),
-            })}
-      >
-        <div className={classes.container}>
-          <div style={{ float: 'left', paddingTop: '8px', paddingLeft: '20px', ...removeSelect }}>
-            {' '}
-            {icon}{' '}
-          </div>
-          <div style={{ padding: '12px 20px 10px 55px', lineBreak: 'auto', ...removeSelect }}>
-            {name}
-          </div>
-        </div>
-        {isSensor && sensorIdx === idx && (
-          <PurePreviewPopup
-            location={location}
-            history={history}
-            sensorReadings={sensorReadings}
-            styleOverride={{ marginTop: 12, marginBottom: 10, position: 'relative', left: 0 }}
-          />
-        )}
+          return (
+            <div
+              className={classes.itemHeader}
+              key={idx}
+              {...(isTouchDevice()
+                ? {
+                    onTouchStart: (e) => {
+                      e.stopPropagation();
+                    },
+                    onTouchEnd: () => handleMouseUp(location, idx),
+                  }
+                : {
+                    onMouseDown: (e) => {
+                      e.stopPropagation();
+                    },
+                    onMouseUp: () => handleMouseUp(location, idx),
+                  })}
+            >
+              <div className={classes.title}>
+                <div
+                  className={clsx(classes.itemIcon, showDetails(location.id) && classes.sensorIcon)}
+                >
+                  {' '}
+                  {icon}{' '}
+                </div>
+                <div className={classes.itemName}>{name}</div>
```

Comment:
It looks like some style adjustments are needed for a soil water content sensor.
Also, when a soil water content sensor is clicked, I think we need to open the edit view. Otherwise, we won't have access to that view.
<img width="393" alt="Screenshot 2023-03-21 at 10 23 10 PM" src="https://user-images.githubusercontent.com/33141219/226809678-4526a45e-6a3e-4cc8-ab07-655556941e2d.png">